packages/components/src/composite/item.tsx

```diff
@@ -20,9 +20,24 @@ export const CompositeItem = forwardRef<
 	WordPressComponentProps< CompositeItemProps, 'button', false >
 >( function CompositeItem( props, ref ) {
 	const context = useCompositeContext();
+
+	// @ts-expect-error The store prop in undocumented and only used by the
+	// legacy compat layer.
+	const storeViaProps = props.store as Ariakit.CompositeStore;
+	const store = storeViaProps ?? ( context.store as Ariakit.CompositeStore );
+
+	// If the active item is not connected, Composite may end up in a state
+	// where none of the items are tabbable. In this case, we force all items to
+	// be tabbable, so that as soon as an item received focus, it becomes active
+	// and Composite goes back to working as expected.
+	const tabbable = Ariakit.useStoreState( store, ( state ) => {
+		return ! store?.item( state?.activeId )?.element?.isConnected;
+	} );
+
```

Comment:
This is the actual fix

Reply:
Just a heads up: it's probably better to check if `activeId` is not `null` beforehand. See https://github.com/ariakit/ariakit/issues/4129#issuecomment-2367928139

Reply:
Thank you, updated!